examples/website/src/shared/teasers/imageTeaser.tsx

```diff
@@ -125,16 +125,20 @@ export function ImageTeaser({
   lead,
   route,
   preTitle
-}: LightTeaserProps) {
+}: LightTeaserProps): JSX.Element {
   const css = useStyle({isSingle})
   return (
     <div className={css(ImageTeaserStyle)}>
       <span className={css(PreTitleStyle, ImageTeaserPreTitleStyle)}>{preTitle}</span>
       <Link route={route}>
         <div className={css(ImageTeaserImageStyle)}>
           <Image
-            src={image.format === 'gif' ? image.url : image.largeURL}
-            alt={image.description || image.caption}
+            src={
+              image?.format === 'gif'
+                ? image?.url
+                : image?.largeURL || 'https://via.placeholder.com/240x240'
+            }
+            alt={image?.description || image?.caption}
```

Comment:
Good workaround. Only, to me looks like there is something wrong higher up. The image prop seems not optional and `ImageData` interface requires `url` and `format` to be set. So should not the workaround be set somewhere else?

Reply:
@faebebin you're right. @bishoymelek will check in the api and editor if the user has selected this teaser style without an image and then throw an error.

@bishoymelek: You might want to convert this PR to a draft until you're ready again.

Reply:
@faebebin  Yes Absolutely correct. discussed it with @roosnic1  and now has validation in the editor and should be on the API level as well afterwords.

Reply:
@roosnic1 I checked my local settings. I have eslint activated, but that is project setting,no? Just, I did not find some setting to change by now. Don't you have those highlightings requiring return type:
![image](https://user-images.githubusercontent.com/21113500/105489316-53d06600-5cb3-11eb-8e58-d66db8367f3d.png)